### DISCUSSION TITLE
Trash: All items disappear after trying to delete a single item

### DISCUSSION THREAD

**Discussion by fuxii** [dani-garcia/vaultwarden#3176] [General]
I have multiple deleted items in my trash. When I select a single item and then choose "Permanently delete selected" it looks like, that all items were deleted (they all disappear). After a refresh of the page, all previously **not** selected items are still there.

First I thought I lost all deleted items, but only the selected items were deleted.

**Comment by kpfleming** [comment]
You'll need to report this in the Bitwarden discussion forums or other locations; Vaultwarden doesn't control the behavior of the client software.

**Reply by fuxii** [reply]
Thanks for the hint. I'm facing this issue in the web vault.

**Reply by jjlin** [reply]
The behavior of the clients can be influenced by the behavior of the server, so unless the issue can be reproduced with the Bitwarden server, it shouldn't be reported to them without more investigation.

**Comment by stefan0xC** [comment]
> I have multiple deleted items in my trash. 

How many? (3 items is not enough to reproduce this behavior)

Not a fix, but you might want to automatically delete them by setting `TRASH_AUTO_DELETE_DAYS=30` (which is by default disabled).

**Reply by fuxii** [reply]
Thanks for your answer. Currently I have 7 items in my trash, so it were 8 before. I didn't changed the default setting, because I didn't want to get my trash emptied automatically.

**Comment by BlackDex** [comment]
I have seen this too sometimes. But not always clear how or when. I also haven't looked into this much my self.
Also, i have seen this on the SaaS, so i think it is a client side issue.

**Reply by fuxii** [reply]
For me it's always reproducible, but it's really not a big deal. I just wanted to mention it here.
